Commands/UserHelper.cs

```diff
@@ -10,7 +10,8 @@ namespace BrackeysBot.Commands
     /// </summary>
     public static class UserHelper
     {
-        public static SettingsTable _settings;
+        public static SettingsTable Settings { get; set; }
```

Comment:
`UserHelper` shouldn't have an instance of the settings, it should probably get the static settings from somewhere else